### ISSUE TITLE
Unable to use deno compile/bundle with Input due to ReferenceError

### ISSUE THREAD

**Issue by juliostanley** [c4spar/deno-cliffy#150]
Hi

I like the library a lot, thanks for working on it. However, I am running into an issue I am not familiar with:

**System**
Windows 10
Deno 1.7.2

---

## Issues

**Scenario 1 (compile)**
```
deno compile -o test --unstable https://deno.land/x/cliffy/examples/prompt/input.ts
./test.exe
```

**Result 1**
```
error: ReferenceError: Cannot access 'mod' before initialization
    at factory (file://$deno$/bundle.js:355:39)
    at file://$deno$/bundle.js:83:14
```

---

**Scenario 2 (bundle)**
```
deno bundle --unstable https://deno.land/x/cliffy/examples/prompt/input.ts test.js
deno run --unstable ./test.js
```

**Result 2**
```
error: Uncaught ReferenceError: Cannot access 'mod' before initialization
    const methodList = Object.entries(mod);
                                      ^
    at factory (file:///C:/Users/..../test.js:355:39)
    at file:///C:/Users/..../test.js:83:14
```

---

## Notes

- The closest issue I've found that resembles it is https://github.com/denoland/deno/issues/8211
- The line that seems to be involved is https://github.com/c4spar/deno-cliffy/blob/598692727a4302ef5c5438f0d018956e4107a6a7/ansi/tty.ts#L69

---

## Questions
- Any thoughts why this may be?

**Comment by c4spar** [comment]
Hey @juliostanley, thx for the detailed issue!

It seems to be a bundle issue with deno/swc. The bundeld code has the wrong order.
I have a smaller reproducable example where i get the same reference error:
```typescript
import { tty } from ".https://deno.land/x/cliffy/ansi/tty.ts";
import { eraseDown } from "https://deno.land/x/cliffy/ansi/ansi_escapes.ts";

eraseDown();
tty.eraseDown();
```

I think the problem is related to the same issue as in denoland/deno#8211. With deno < 1.5.0 it works, with >= 1.5.0 it does not.

**Comment by c4spar** [comment]
I have created a deno issue: https://github.com/denoland/deno/issues/9467

**Comment by juliostanley** [comment]
@c4spar thank you for checking. Will keep an eye on updates.

**Comment by c4spar** [comment]
Hi @juliostanley, the issue is fixed in deno 1.8.0.

**Comment by juliostanley** [comment]
Hi @c4spar, it works. Thank you.
